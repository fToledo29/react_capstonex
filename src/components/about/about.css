.about-title {
	text-align: center;
}

.about-content {
	margin-left: auto;
	margin-right: auto;
	padding: 2rem;
	width: 80%;
	text-align: justify;
}

