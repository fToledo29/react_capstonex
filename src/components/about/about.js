import React from 'react'

export const About = () => {
	return (
		<div>

			<section>
				<h3>Company's history</h3>
				<p>
					Incididunt enim tempor occaecat proident ea quis do consequat esse. Culpa in commodo adipisicing quis aute ad sunt quis sunt sunt commodo. Cillum fugiat mollit nulla laborum irure culpa et commodo duis eiusmod adipisicing labore consectetur consectetur. Et exercitation mollit id sunt quis ea commodo voluptate ut irure. Occaecat id consectetur consectetur aute aute minim nisi sunt. Sit aliquip cillum enim in aliquip sunt ad qui deserunt culpa culpa nulla fugiat.

					Aute eu ullamco cillum tempor esse ipsum id proident esse et dolor incididunt. Cupidatat est eiusmod dolore nisi velit minim consequat est minim. Enim labore nostrud ea pariatur nostrud eu minim. Amet id excepteur ullamco ad dolore aliqua minim irure qui laboris.
				</p>
			</section>
		</div>
	)
}
