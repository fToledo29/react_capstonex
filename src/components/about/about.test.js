import React from 'react';
import 'enzyme-to-json';
import { shallow } from 'enzyme';
import { About } from './about';

describe('About', () => {

	let wrapper;

	beforeAll(() => {

		wrapper = shallow(<About />);

	});

	it('About Renders correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});

	it('Show Title', () => {
		expect(wrapper.find('.about-title').text()).toContain(`Company's history`);
	});
});
