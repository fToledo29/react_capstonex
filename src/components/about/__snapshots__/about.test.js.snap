// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About About Renders correctly 1`] = `
<div>
  <h2
    className="about-title"
  >
    Company's history
  </h2>
  <div
    className="about-content"
  >
    <h3>
      This content is just for testing..
    </h3>
    <span>
      <p>
        Incididunt enim tempor occaecat proident ea quis do consequat esse. Culpa in commodo adipisicing quis aute ad sunt quis sunt sunt commodo. Cillum fugiat mollit nulla laborum irure culpa et commodo duis eiusmod adipisicing labore consectetur consectetur. Et exercitation mollit id sunt quis ea commodo voluptate ut irure. Occaecat id consectetur consectetur aute aute minim nisi sunt. Sit aliquip cillum enim in aliquip sunt ad qui deserunt culpa culpa nulla fugiat.
      </p>
      <p>
        Aute eu ullamco cillum tempor esse ipsum id proident esse et dolor incididunt. Cupidatat est eiusmod dolore nisi velit minim consequat est minim. Enim labore nostrud ea pariatur nostrud eu minim. Amet id excepteur ullamco ad dolore aliqua minim irure qui laboris.
      </p>
      <p>
        Officia irure eiusmod duis labore labore ad voluptate et sint ex. Nulla mollit incididunt fugiat ullamco exercitation. Ea mollit reprehenderit exercitation irure eiusmod incididunt est excepteur irure amet labore sunt elit cillum. Nostrud qui tempor ea in dolore et minim irure labore ipsum non esse. Minim ex nostrud cillum ullamco elit. Nulla exercitation adipisicing fugiat amet sit cupidatat nostrud deserunt qui eu elit tempor fugiat. Et dolore adipisicing minim veniam sunt consectetur esse proident velit pariatur culpa. Excepteur qui excepteur ea dolore tempor. In sint velit magna ea consectetur velit pariatur non amet qui ipsum. Amet minim reprehenderit ex labore laborum mollit sint sint ut. Labore voluptate velit incididunt Lorem elit excepteur est excepteur quis cillum velit. Pariatur magna ut incididunt non sunt et commodo officia consequat mollit do ex.
      </p>
      <p>
        Mollit cillum nostrud voluptate consectetur consectetur. Eiusmod in aute deserunt proident proident non eu do laborum eiusmod incididunt incididunt quis. Non commodo dolor exercitation sit quis id nostrud ipsum in. Nulla eiusmod sit nisi mollit. Labore ipsum id elit laboris id Lorem incididunt reprehenderit dolore dolor nulla velit. Ex ipsum dolor qui pariatur nisi incididunt anim laboris. Eiusmod anim cillum minim sint ullamco in incididunt et mollit. Cupidatat mollit pariatur pariatur adipisicing.
      </p>
      <p>
        Aliqua nisi adipisicing tempor cillum sit exercitation dolore laboris nostrud magna in. Ex amet occaecat aliqua consequat reprehenderit. Sit consectetur id magna eiusmod. Minim aliquip veniam qui et reprehenderit ex reprehenderit sint voluptate commodo. Quis incididunt dolor exercitation magna duis amet reprehenderit ex aliquip irure proident. Eiusmod dolore deserunt enim laboris laboris aute eiusmod minim. Anim non et commodo ea nulla esse dolore fugiat eu est incididunt nisi nisi. Pariatur in aliqua nisi consequat consectetur veniam ipsum anim aute incididunt irure. Occaecat reprehenderit labore commodo nostrud elit proident sunt velit dolor anim laboris consequat. Proident qui veniam consectetur voluptate eiusmod veniam ad aliquip et dolore esse amet. Non ipsum excepteur velit aliqua irure reprehenderit labore ea culpa labore in. Labore ea cupidatat ad ullamco tempor sint ullamco. Qui occaecat do ipsum nostrud deserunt eu eiusmod qui nostrud Lorem dolore. Velit consectetur do aliqua mollit elit exercitation. Nulla laborum occaecat ex dolor excepteur irure dolor magna. Commodo labore eiusmod culpa incididunt veniam enim aliqua. Culpa adipisicing minim voluptate duis laborum reprehenderit.
      </p>
      <p>
        Ad velit amet incididunt veniam tempor elit elit enim. Voluptate elit qui id consectetur laboris aliquip. Sit laborum magna est consequat mollit. Ex esse non commodo minim aliquip minim. Enim labore nulla occaecat ipsum officia ad voluptate do ut minim mollit pariatur nulla sunt. Enim laboris eu sit aute commodo laboris Lorem cupidatat sit velit dolor incididunt. Amet in incididunt exercitation minim aliqua. Enim dolore nulla cillum nostrud sint esse cupidatat veniam. Aliquip fugiat id enim tempor labore consectetur ipsum tempor. Officia veniam ad eiusmod tempor ipsum commodo ullamco consectetur sunt Lorem consectetur incididunt. Aliqua proident consequat incididunt sit aliquip excepteur ex. Commodo magna Lorem dolore enim Lorem est nisi nulla excepteur ex. Tempor nostrud laborum amet non est ullamco sit laborum do excepteur. Labore ex anim exercitation Lorem eiusmod pariatur mollit quis incididunt fugiat anim minim excepteur ad.
      </p>
    </span>
  </div>
</div>
`;
